### ISSUE TITLE
Test checking output string fails in localized Windows

### ISSUE THREAD

**Issue by tautologico** [mathnet/mathnet-symbolics#20]
Running tests on a Windows configured to a Brazil locale fails one test because the system uses comma for separating the integer and fractionary parts in a floating-point number, instead of a dot. Test output: 

```
1) Failed : Tests.Constant Expressions
  String lengths are both 5. Strings differ at index 1.
  Expected: "3.2*x"
  But was:  "3,2*x"
```

**Comment by FoggyFinder** [comment]
well, it's easy to fix:

https://github.com/mathnet/mathnet-symbolics/pull/21

**Comment by cdrnet** [comment]
I wonder whether it is wise to allow such culture variation in this formal mathematical language. This would make it rather hard to support expression lists, e.g. to describe a matrix. Instead we could enforce the invariant culture everywhere (I'm actually surprised we're not doing this already).

**Comment by tautologico** [comment]
It's probably better to have invariant syntax anyway, as programming languages do.

**Comment by cdrnet** [comment]
@FoggyFinder what are your thoughts, would this work for you as well?

**Comment by FoggyFinder** [comment]
@cdrnet I agree, it's nice idea

**Comment by tautologico** [comment]
I believe this can be closed now, right?

**Comment by cdrnet** [comment]
Invariant culture is now enforced, released in v0.14.0.
